erpnext/selling/doctype/customer/customer.json

```diff
@@ -188,7 +188,7 @@
    "fieldtype": "Link",
    "in_standard_filter": 1,
    "label": "Account Manager",
-   "options": "User"
+   "options": "Sales Person"
```

Comment:
Lets write a patch to unset any invalid links, since we are changing the link type, old data might link to users, we need to make sure any customer with account manager field does not reference a sales person should be unset, otherwise the user will get errors when trying to save the customer